types/universal-router/index.d.ts

```diff
@@ -1,68 +1,118 @@
-// Type definitions for universal-router 1.2
+// Type definitions for universal-router 6.0
 // Project: https://github.com/kriasoft/universal-router
 // Definitions by: Jack Moore <https://github.com/jtmthf>
+//                 Tomek Łaziuk <https://github.com/tlaziuk>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
 
 /**
- * Params is a key/value object that represents extracted URL paramters. Each
- * URL parameter resolves to a string.
+ * Params is a key/value object that represents extracted URL paramters.
+ * Each URL parameter resolves to a string.
  */
 export interface Params {
-  [key: string]: string;
+    [_: string]: any;
```

Comment:
Could I ask, why do you replace named variables with underscores? It means nothing for Typescript, but I think it would be good to name object keys to be consistent with other packages.

Reply:
I've found `_` to be more pleasant and readable to me as the index prop name.

Reply:
Well, I suppose it's a matter of taste :)